yb-voyager/cmd/live_migration.go

```diff
@@ -45,6 +46,7 @@ var END_OF_QUEUE_SEGMENT_EVENT = &tgtdb.Event{Op: "end_of_source_queue_segment"}
 var FLUSH_BATCH_EVENT = &tgtdb.Event{Op: "flush_batch"}
 var eventQueue *EventQueue
 var statsReporter *reporter.StreamImportStatsReporter
+var tgtDbQueryMutex sync.Mutex
```

Comment:
This is an incorrect place for adding this mutex. If really required, it should be inside the `tdb`.

Reply:
I added it because tdb.Query uses the single connection in each targetDB, not the conn pool. 

I agree with your suggestion. it makes more sense in tdb. Should I make all the implementations of tdb.Query/Exec/etc use a mutex?